### DISCUSSION TITLE
storyboard

### DISCUSSION THREAD

**Discussion by karmatradeDev** [showtime-xyz/showtime-frontend#1255] [Q&A]
hi, i really like how this app is structured I usually build react apps but react native is honestly kinda different. I would like to do story-driven design which is building the storyboard components first and then build the screens and features. I saw that the storyboard is set up a little differently in solito would you mind explaining how you did it and why?

**Comment by alantoa** [comment]
Hey! @karmatradeDev 
Can you elaborate on which step, I don't quite understand what you mean, thanks!
